<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TQtSettings</class>
 <widget class="QDialog" name="TQtSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>633</width>
    <height>372</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>629</width>
    <height>330</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Settings</string>
  </property>
  <property name="locale">
   <locale language="English" country="Europe"/>
  </property>
  <property name="modal">
   <bool>false</bool>
  </property>
  <widget class="QScrollArea" name="scrollArea">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>631</width>
     <height>371</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>629</width>
      <height>369</height>
     </rect>
    </property>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>330</y>
       <width>601</width>
       <height>32</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>411</width>
       <height>32</height>
      </size>
     </property>
     <property name="locale">
      <locale language="English" country="Europe"/>
     </property>
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_logLevel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Log steps</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogAll">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>10</y>
       <width>51</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>All</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogError">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>10</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Error</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogInfo">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>10</y>
       <width>49</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Info</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogDebug">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>10</y>
       <width>71</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Debug</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogWarning">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>10</y>
       <width>75</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Warning</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogTrace">
     <property name="geometry">
      <rect>
       <x>330</x>
       <y>10</y>
       <width>61</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Trace</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_LogProtocol">
     <property name="geometry">
      <rect>
       <x>490</x>
       <y>10</y>
       <width>81</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>Protocol</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_logFile">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>70</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Logfile</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_logFile">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>70</y>
       <width>370</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>370</width>
       <height>36</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="kiconbutton_reset">
     <property name="geometry">
      <rect>
       <x>520</x>
       <y>70</y>
       <width>44</width>
       <height>38</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>44</width>
       <height>38</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="tpanel.qrc">
       <normaloff>:/images/reset.png</normaloff>:/images/reset.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QToolButton" name="kiconbutton_logFile">
     <property name="geometry">
      <rect>
       <x>570</x>
       <y>70</y>
       <width>44</width>
       <height>38</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>44</width>
       <height>38</height>
      </size>
     </property>
     <property name="icon">
      <iconset resource="tpanel.qrc">
       <normaloff>:/images/fileopen.png</normaloff>:/images/fileopen.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>32</width>
       <height>32</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_Controller">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>130</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Controller</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_Controller">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>120</y>
       <width>471</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>36</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_Port">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>170</y>
       <width>121</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Network port</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Port">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>141</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>36</height>
      </size>
     </property>
     <property name="minimum">
      <number>1024</number>
     </property>
     <property name="maximum">
      <number>32767</number>
     </property>
     <property name="value">
      <number>1397</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_Channel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>210</y>
       <width>120</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>120</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Channel number</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="spinBox_Channel">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>200</y>
       <width>141</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>36</height>
      </size>
     </property>
     <property name="minimum">
      <number>10001</number>
     </property>
     <property name="maximum">
      <number>10999</number>
     </property>
     <property name="value">
      <number>10001</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_PType">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>250</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Panel type</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="lineEdit_PType">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>240</y>
       <width>471</width>
       <height>36</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>36</height>
      </size>
     </property>
    </widget>
    <widget class="QLabel" name="label_PType_2">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>300</y>
       <width>111</width>
       <height>22</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>111</width>
       <height>22</height>
      </size>
     </property>
     <property name="text">
      <string>Scaling</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Scale">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>300</y>
       <width>141</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>141</width>
       <height>28</height>
      </size>
     </property>
     <property name="text">
      <string>Scale to fit</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_logFormats">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>70</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Log formats</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Profiling">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>40</y>
       <width>91</width>
       <height>28</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>91</width>
       <height>28</height>
      </size>
     </property>
     <property name="toolTip">
      <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Adds profiling time to logfile.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
     </property>
     <property name="text">
      <string>Profiling</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="checkBox_Format">
     <property name="geometry">
      <rect>
       <x>270</x>
       <y>40</y>
       <width>131</width>
       <height>26</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>121</width>
       <height>26</height>
      </size>
     </property>
     <property name="text">
      <string>Long format</string>
     </property>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources>
  <include location="tpanel.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>TQtSettings</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>TQtSettings</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
